sandbox.html

```diff
@@ -70,14 +70,24 @@
 
     const cadenzaClient = cadenza(location.origin + '/trunk', { iframe: 'iframe', debug: true });
     const actionHandlers = {
-      show ({ embeddingTargetId, hideMainHeaderAndFooter, hideWorkbookToolBar, jasperReportAsPdf, highlightGlobalId }) {
+      show ({ embeddingTargetId, expandNavigator, hideMainHeaderAndFooter, hideWorkbookToolBar, jasperReportAsPdf, highlightGlobalId }) {
         cadenzaClient.show(embeddingTargetId, {
+          expandNavigator: (expandNavigator === 'on'),
           hideMainHeaderAndFooter: (hideMainHeaderAndFooter === 'on'),
           hideWorkbookToolBar: (hideWorkbookToolBar === 'on'),
           highlightGlobalId,
           ...(jasperReportAsPdf === 'on' && { mediaType: 'application/pdf' }),
         });
       },
+      showPage ({ page, expandNavigator, highlightGlobalId }) {
+        cadenzaClient.show({ page: page }, {
+          expandNavigator: (expandNavigator === 'on'),
+          highlightGlobalId
```

Comment:
These are 2 different things, but closely related, hence the possible confusion. As the name implies, `expandNavigator` expands the left navigation panel, making the navigation tree visible. 

highlightGlobalId selects the given item from the tree and also expands the tree item (if it can be expanded). But has nothing to do with the tree panel being visible.

Reply:
AFAIK, _this_ is the discussion. that's why we do a review :)

Reply:
I don't know whether or not this is the discussion for naming. But I will clearify that with @jkissel . For naming I would propose "showNavigator" instead of "expandNavigator".

Reply:
I think `showNavigator()` would indeed conflict a bit with what we're used from `show()` / `showMap()` so I'd propose to stick with `expandNavigator()`.

Reply:
OK :+1: